kstatemachine/src/commonMain/kotlin/ru/nsk/kstatemachine/TransitionStateApi.kt

```diff
@@ -61,9 +69,10 @@ inline fun <reified E : Event> TransitionStateApi.transition(
  */
 inline fun <reified E : Event> TransitionStateApi.transition(
     name: String? = null,
+    metaInfo: TransitionMetaInfo? = null,
```

Comment:
I think for transition builder functions that provide lambda as last argument, 
metaInfo should be set there as a builder property. not as function argument.
This should give more nice DSL view.